CONTRIBUTING.md

```diff
@@ -18,9 +18,9 @@ propose changes to this document in a pull request.
 but before doing that please read the notes below and include as many details as
 possible with your report. If you can, please include:
   * The version of Electron you are using
-  * The operating system you are using
-  * If applicable, what you were doing when the issue arose and what you
-  expected to happen
+  * The specifications of the operating system you are using
```

Comment:
@skarthikeya All we normally need is which OS so we can determine windows/macos/linux so if we need to reproduce the issue we know which OS to try.  The specs of the machine don't affect it 99.99% of the time